pnpm/src/cmd/recursive.ts

```diff
@@ -77,11 +77,11 @@ and must recompile all your C++ addons with the new binary.',
 
         list: [
           {
-            description: 'Continues executing other tasks even if a task threw an error.',
+            description: 'Continues executing other tasks even if a task threw an error',
             name: '--no-bail',
           },
           {
-            description: 'Set the maximum number of concurrency. Default is 4. For unlimited concurrency use Infinity.',
+            description: 'Set the maximum number of concurrency. Default is 4. For unlimited concurrency use Infinity',
```

Comment:
When there are two sentences. I believe we keep the dot.

Reply:
Soory about this, I made the line break a condition for whether to add dot, I'll change it again